config/manifests/bases/kubernetes-image-puller-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Seamless Upgrades
+    categories: Developer Tools
+    certified: "false"
+    containerImage: quay.io/eclipse/kubernetes-image-puller-operator:0.0.9
+    createdAt: ""
+    description: Create and manage kubernetes-image-puller instances.
+    repository: https://github.com/che-incubator/kubernetes-image-puller-operator
+    support: ""
+  name: kubernetes-imagepuller-operator.v0.0.0
```

Comment:
I think this should be `kubernetes-imagepuller-operator.v0.0.9`

Reply:
This file, in general is a template. We shouldn't use this file directly. This file should be used with kustomize command. Kustomize binary automatically patch deployment yamls. So this value will be changed from v0.0.0 to the version defined in the Makefile, or version defined in the deployment command. For example when your're doing deploy command:

```bash
$ make deploy -s
``` 

Kustomize will change value from 0.0.0 to "VERSION" value from Makefile. New operator-sdk doesn't suppose that developer applies yamls from config folder directly.

But good catch, on the release we should update Makefile version value too. I will handle it. Thanks.

Reply:
ah I see, thank you